.github/workflows/build+test.yml

```diff
@@ -22,5 +22,11 @@ jobs:
           java-version: "${{env.java_version}}"
           distribution: temurin
           cache: gradle # caches dependencies (https://github.com/actions/setup-java#caching-packages-dependencies)
-      - name: Build project and run unit tests
-        run: ./gradlew build --no-daemon
+      - name: Build project and run unit tests (local flavor)
```

Comment:
What does local flavor mean? :)

Reply:
"Flavor" is a standard concept in the Android world understood by the [Android Gradle Plugin](https://developer.android.com/studio/releases/gradle-plugin). A flavor and build type (release or debug) make up a [build variant](https://developer.android.com/studio/build/build-variants). In our app we have a flavor for each of the networks (stagenet, testnet, mainnet) and "local" for running all the supporting services locally. There's also one called "mock" which seems to be some variant of stagenet (it isn't clear what it's for).

The only difference between the flavors is the configuration defined in the `productFlavors` section in `./app/build.gradle`. I.e. stuff like the Wallet Proxy URL, whether to show a CCD drop button etc.

I figured that the test coverage is the same for all the flavors and that the local one is the most "neutral" one of them.

Reply:
Thanks for the elaborate and concise answer. I learned something new and the day has barely started :)